packages/lesswrong/components/tagging/subforums/SubforumSubscribeSection.tsx

```diff
@@ -59,8 +59,8 @@ const SubforumSubscribeSection = ({
       captureEvent('subforumSubscribeClicked', {tagId: tag._id});
 
       if (currentUser) {
-        await subforumMembershipMutation({variables: {tagId: tag._id, member: true}});
         joinCallback();
+        await subforumMembershipMutation({variables: {tagId: tag._id, member: true}});
```

Comment:
You can basically ignore this file for the purposes of review. The only real change here is to call the join callback before completing the mutation. This doesn't actually achieve much atm, but at some point I'm planning on making it eagerly show the notification menu to avoid the 2s hang time.